sources/javascripts/concat/base.js

```diff
@@ -569,6 +569,18 @@
     }
   };
 
+  // ===========================================================================
+  // Opens product admin view on product image click
+  // ===========================================================================
+
+  var handleProductImageClick = function(url) {
```

Comment:
Let's rename as it actually is: `id` or slug not `url`.

```suggestion
  var handleProductImageClick = function(product_id) {
```